shadow-cljs.edn

```diff
@@ -17,11 +17,16 @@
                              :cljify calva.js-utils/cljify
                              :prettyPrint calva.pprint.printer/pretty-print-js-bridge
                              :parseEdn calva.parse/parse-edn-js-bridge
-                             :parseForms calva.parse/parse-forms-js-bridge}
+                             :parseForms calva.parse/parse-forms-js-bridge
+                             :activateLsp calva.lsp/activate
+                             :deactivateLsp calva.lsp/deactivate
+                             :getReferences calva.lsp/get-references
+                             :getDocumentSymbols calva.lsp/get-document-symbols
+                             :LSP_CLIENT_KEY calva.lsp/client-key}
                  :output-to "out/cljs-lib/cljs-lib.js"}
                 :test
                 {:target    :node-test
                  :output-to "out/cljs-lib/test/cljs-lib-tests.js"
-                 :ns-regexp ".*"
+                 :ns-regexp "-test$"
```

Comment:
With this set to `.*` as it was before, we'd get an error during the release script like "cannot find module 'vscode'" since adding the lsp.cljs file. With this regexp, we get the same number of tests and assertions as we currently do on dev.

Reply:
With regard to the above, I think that is because I moved a test from the pretty-print function to its own test namespace in order to make sure it was run.

Reply:
I really want to retain the possibility to have the tests inlined in the functions. This change looks like it is closing that. option. Can we factor te code such that the vscode using modules have something in their namespace names that we can exclude?

Reply:
We can still inline tests with this regexp. I don't know exactly how it works, but if you run the tests you'll notice the inline tests are run. I just added the test I moved out of the `calva.pprint.pprinter` namespace back into that namespace and re-ran the tests and verifeed that it ran, along with other inline tests in other namespaces. So this seems to work for what we need.

Reply:
I believe what it does it compile namespaces that either end in `-test` _or_ that have inline tests.